src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_transformers.py

```diff
@@ -62,6 +63,34 @@ def transform_entity_query_output(result):
     return new_results
 
 
+def transform_entities_result(result):
+    new_items = []
+    for entity in result.items:
+        new_items.append(transform_entity_result(entity))
+    return {
+        'items': new_items,
+        'nextMarker': result.next_marker
+    }
```

Comment:
Can you reuse the original result object and simply replace items? If the server decides to start sending new information, this will not be able to get that.

Reply:
Just for the result object, not for individual entities as well?

Reply:
Same for the entities, only replace the part being fixed, which is the entity value.

Reply:
In that case the only update will be on the individual entities, i.e.:

```python
def transform_entities_result(result):
    for entity in result.items:
        transform_entity_result(entity)
    return result


def transform_entity_result(entity):
    for key in entity.keys():
        property = entity[key]
        if hasattr(property, 'value') and isinstance(property.value, bytes):
            property.value = base64.b64encode(property.value).decode()
    return entity
```